Dockerfile

```diff
@@ -0,0 +1,65 @@
+FROM node:alpine AS vendors
+
+COPY . /srv/umap
+
+WORKDIR /srv/umap
+
+RUN npm install
+
+RUN npm run vendors
+
+FROM python:3.11-slim
+
+ENV PYTHONUNBUFFERED=1 \
+    PORT=8000
+
+RUN mkdir -p /srv/umap/uploads
+
+COPY . /srv/umap
+
+COPY --from=vendors /srv/umap/umap/static/umap/vendors /srv/umap/umap/static/umap/vendors
+
+WORKDIR /srv/umap
+
+RUN apt-get update && \
```

Comment:
Suggestion: move that whole `RUN apt-get...` block so that it's just after the `FROM` line; that way it won't be executed again every single time a file is changed

Reply:
Should be good, thanks :)